nipoppy/data/examples/sample_pipelines/freesurfer_stats_and_qc-0.1.0/invocation.json

```diff
@@ -0,0 +1,11 @@
+{
+    "script_path": "[[NIPOPPY_DPATH_PIPELINES]]/freesurfer_stats_and_qc-0.1.0/script.py",
+    "input_dir_path": "[[NIPOPPY_DPATH_PIPELINE_OUTPUT]]",
+    "output_stats_file_path": "[[NIPOPPY_DPATH_PIPELINE_IDP]]/freesurfer_stats-thickness.tsv",
+    "output_qc_file_path": "[[NIPOPPY_DPATH_PIPELINE_IDP]]/freesurfer_qc.tsv",
+    "mode": "multi",
+    "container": "[[NIPOPPY_CONTAINER_COMMAND]] [[NIPOPPY_FPATH_CONTAINER]]",
+    "aseg_args": " --skip",
+    "aparc_args": " --skip --meas=thickness",
```

Comment:
i think `meas` needs to be a tag as well. `thickness` and `surface-area` are both common and would be easier to parse as a key-val